README.md

```diff
@@ -91,6 +91,33 @@ var observedObject = jsonPatcherProxy.observe(true, function(patch) {
 observedObject.firstName = "Albert";
 ```
 
+### Multiple instances collisions
+
+If you have two or more instances of JSONPatcherProxy, exchanging values of type **object** among them might cause unexpected issues.
+
+Consider
+```js
+var myObj = { contactDetails: { phoneNumbers: [ { number:"555-123" }] } };
+var jsonPatcherProxyOne = new JSONPatcherProxy( myObj );
+var observedObjectOne = jsonPatcherProxyOne.observe(true);
+
+var jsonPatcherProxyTwo = new JSONPatcherProxy( myObj );
+var observedObjectTwo = jsonPatcherProxyTwo.observe(true);
+
+// this deep clones observedObjectOne.contactDetails before setting it to observedObjectTwo.newContactDetails, to avoid double proxification.
+observedObjectTwo.newContactDetails = observedObjectOne.contactDetails;
+```
+In this case, you might expect mutating `observedObjectOne.contactDetails` would be equivalent to mutating `observedObjectTwo.newContactDetails`, but it's not. because setting proxies coming from other instances of JSONPatcherProxies will mean deep cloning to remove the originally watching proxy and proxify the tree again.
+
+```js
+var myObj = { firstName:"Joachim", lastName:"Wester", contactDetails: { phoneNumbers: [ { number:"555-123" }] } };
+var jsonPatcherProxy = new JSONPatcherProxy( myObj );
+var observedObject = jsonPatcherProxy.observe(true, function(patch) {
+    // patch == { op:"replace", path="/firstName", value:"Albert"}
+});
+observedObject.firstName = "Albert";
+```
+
```

Comment:
I don't really understand the snippet between lines 113-118. How is it related to the paragraph above?